src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ExcelAdaptorServiceImpl.java

```diff
@@ -283,7 +283,7 @@ private Object getCellValue(Cell cell) {
         }
         switch (cell.getCellType()) {
             case STRING:
-                return cell.getStringCellValue();
+                return cell.getStringCellValue().trim();
```

Comment:
Is there any possibility that this value can be blank, this inducing NPE?

Reply:
We are doing a null check just before, so it shouldn't
```
if (isNull(cell)) {
     return null;
}
switch (cell.getCellType()) {
  case STRING:
     return cell.getStringCellValue().trim();
```